x-pack/plugins/cases/server/client/attachments/get.ts

```diff
@@ -112,45 +111,26 @@ export async function find(
     const { filter: authorizationFilter, ensureSavedObjectsAreAuthorized } =
       await authorization.getAuthorizationFilter(Operations.findComments);
 
-    const id = caseID;
-    const { filter, ...queryWithoutFilter } = queryParams ?? {};
-
-    // if the fields property was defined, make sure we include the 'owner' field in the response
-    const fields = includeFieldsRequiredForAuthentication(queryWithoutFilter.fields);
-
-    // combine any passed in filter property and the filter for the appropriate owner
-    const combinedFilter = combineFilters([stringToKueryNode(filter), authorizationFilter]);
-
-    const args = queryParams
-      ? {
-          caseService,
-          unsecuredSavedObjectsClient,
-          id,
-          options: {
-            // We need this because the default behavior of getAllCaseComments is to return all the comments
-            // unless the page and/or perPage is specified. Since we're spreading the query after the request can
-            // still override this behavior.
-            page: DEFAULT_PAGE,
-            perPage: DEFAULT_PER_PAGE,
-            sortField: 'created_at',
-            filter: combinedFilter,
-            ...queryWithoutFilter,
-            fields,
-          },
-        }
-      : {
-          caseService,
-          unsecuredSavedObjectsClient,
-          id,
-          options: {
-            page: DEFAULT_PAGE,
-            perPage: DEFAULT_PER_PAGE,
-            sortField: 'created_at',
-            filter: combinedFilter,
-          },
-        };
-
-    const theComments = await caseService.getAllCaseComments(args);
+    const filter = combineFilters([
+      buildFilter({
+        filters: [CommentType.user],
```

Comment:
I can't remember if we discussed this. Just want to confirm that the find api will only retrieve user comments going forward?

Reply:
I remember talking about it offline with @cnasikas so fwiw this is deliberate 😄

Reply:
Yes as the API will be public it is better to be as strict as possible and relax in the future if needed.